documentation/modules/deploying/proc-deploy-migrate-kraft.adoc

```diff
@@ -0,0 +1,256 @@
++
+[source,shell]
+----
+kubectl get pods -n my-project
+----
++
+.Output shows nodes in broker and controller node pools
+[source,shell]
+----
+NAME                     READY  STATUS   RESTARTS
+my-cluster-kafka-0       1/1    Running  0
+my-cluster-kafka-1       1/1    Running  0
+my-cluster-kafka-2       1/1    Running  0
+my-cluster-controller-3  1/1    Running  0
+my-cluster-controller-4  1/1    Running  0
+my-cluster-controller-5  1/1    Running  0
+# ...
+----
++
+Here, the brokers have the name `my-cluster-kafka` as they were contained in a node pool named `kafka` when migrating the cluster to use node pools.
+
+. Check the status of the migration:
++
+[source,shell]
+----
+kubectl get kafka my-cluster -n my-project -w
+----
++
+.Updates to the metadata state
+[source,shell]
+----
+NAME        ...  METADATA STATE
+my-cluster  ...  Zookeeper
+my-cluster  ...  KRaftMigration
+my-cluster  ...  KRaftDualWriting
+my-cluster  ...  KRaftPostMigration
+----
++
+`METADATA STATE` shows the mechanism used to manage Kafka metadata and coordinate operations.
+At the start of the migration this is `ZooKeeper`.
++
+--
+* `ZooKeeper` is the initial state when metadata is only stored in ZooKeeper.
+* `KRaftMigration` is the state when the migration is in progress and brokers are rolled to register with the controllers.
+The migration can take some time at this point depending on the number of topics and partitions in the cluster. 
+* `KRaftDualWriting` is the state when the Kafka cluster is working as a KRaft cluster, 
+but controllers are managing metadata in Kafka and ZooKeeper. 
+Brokers are rolled a second time to remove the ZooKeeper configuration and migration annotation from the `Kafka` resource.
+* `KRaftPostMigration` is the state when KRaft mode is enabled for brokers and there is no ZooKeeper involvement. 
+Controllers are still connected to ZooKeeper.
+You can xref:proc-deploy-migrate-kraft-rollback-{context}[roll back from this point].
+--
++
+The migration status is also represented in the `status.kafkaMetadataState` property of the `Kafka` resource. 
+
+. Enable KRaft in the `Kafka` resource configuration by setting the `strimzi.io/kraft` annotation to `enabled`:
++
+[source,shell]
+----
+kubectl annotate kafka my-cluster strimzi.io/kraft: enabled
+----
++
+.Enabling KRaft migration
+[source,yaml,subs="+attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+  namespace: my-project
+  annotations:
+    strimzi.io/kraft: enabled 
+# ...
+----
++
+WARNING: Rollback cannot be performed after enabling KRaft.
+
+. Check the status of the move to full KRaft mode:
++
+[source,shell]
+----
+kubectl get kafka my-cluster -n my-project -w
+----
++
+.Updates to the metadata state
+[source,shell]
+----
+NAME        ...  METADATA STATE
+my-cluster  ...  Zookeeper
+my-cluster  ...  KRaftMigration
+my-cluster  ...  KRaftDualWriting
+my-cluster  ...  KRaftPostMigration
+my-cluster  ...  KRaft             
```

Comment:
```suggestion
my-cluster  ...  PreKRaft             
my-cluster  ...  KRaft             
```

I added the new PreKRaft state which is there since the ZooKeeper deletion was implemented.